lib/types/src/circleci.ts

```diff
@@ -7,6 +7,7 @@ export type JobConfig = {
   requires?: string[]
   filters?: { branches?: { only?: string; ignore?: string }; tags?: { only?: string } }
   executor?: string
+  [parameter: string]: unknown
```

Comment:
**question:** what is this for?

Reply:
Any [parameters](https://circleci.com/docs/configuration-reference/#parameters-job) you've specified in a job definition can be passed to the job. These parameters can be any string (excluding some reserved words and only with valid characters) which we represent with an [index signature](https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures). For example, the Tool Kit `heroku-provision` job now has takes [two new parameters](https://github.com/Financial-Times/dotcom-tool-kit/blob/9f3c4be57255fd39b87aed801218cc00c7769c65/orb/src/jobs/heroku-provision.yml#L5-L10), which we use to [pass settings from your options](https://github.com/Financial-Times/dotcom-tool-kit/blob/9f3c4be57255fd39b87aed801218cc00c7769c65/plugins/backend-serverless-app/src/index.ts#L23-L24).